src/components/dialog/dialog.scss

```
@@ -120,4 +120,7 @@ slot[name='button'] {
     slot[name='button'] {
         flex-direction: column-reverse;
     }
+    .mdc-dialog__actions {
+        padding: 1rem 1.5rem 1.5rem;
```

I think this shorthand is horrible to read. I think the 4 and 2 value one is bad enough, but with this one, I never have _any_ clue what value will be applied where 😭

```suggestion
        padding: 1rem 1.5rem 1.5rem 1.5rem;
```